.gitattributes

```diff
@@ -1,11 +1,9 @@
 # Normalize line endings in the repository
 * text=auto
-
 # These file types should always be considered binary
 *.png  binary
 *.jpg  binary
 *.rtf  binary
 *.pdf  binary
-*.onnx binary
```

Comment:
Do we need to remove `.onnx` extension from this file?

Reply:
removed